### PR TITLE
PK swap implemented

### PR DESCRIPTION
**Pull request by emsimpson92**

### CODE REVIEW

File: sim/rogue/poisons.go [hunk 1]
```diff
@@ -83,6 +83,17 @@ func (rogue *Rogue) applyPoisons() {
...
 	rogue.applyAtrophicPoison()
 	rogue.applyNumbingPoison()
+
+	if rogue.Options.PkSwap && rogue.HasRune(proto.RogueRune_RunePoisonedKnife) {
+		rogue.RegisterAura(core.Aura{
+			Label:    "Apply Sebacious on pull (PK Swap)",
+			Duration: core.NeverExpires,
+			OnReset: func(aura *core.Aura, sim *core.Simulation) {
+				rogue.SebaciousPoison[1].Cast(sim, sim.GetTargetUnit(0))
+				aura.Activate(sim)
```
Comment (kayla-glick) [inline sim/rogue/poisons.go:93]: I don't think you'd actually need to activate the aura here

File: ui/rogue/inputs.ts [hunk 2]
```diff
@@ -11,4 +11,10 @@ export const HonorOfThievesCritRate = InputHelpers.makeSpecOptionsNumberInput<Sp
 	fieldName: 'honorAmongThievesCritRate',
 	label: 'Honor Among Thieves Crit Rate',
 	labelTooltip: 'Number of crits other group members generate within 100 seconds',
+});
+
+export const pkSwap = InputHelpers.makeSpecOptionsBooleanInput<Spec.SpecRogue>({
+	fieldName: 'pkSwap',
+	label: 'PK Swap',
+	labelTooltip: 'Apply sebacious poison to the boss on pull',
```
Comment (kayla-glick) [inline ui/rogue/inputs.ts:19]: Can we make this more descriptive, like "Poisoned Knife Swap"?

File: ui/tank_rogue/inputs.ts [hunk 3]
```diff
@@ -11,4 +11,10 @@ export const HonorOfThievesCritRate = InputHelpers.makeSpecOptionsNumberInput<Sp
 	fieldName: 'honorAmongThievesCritRate',
 	label: 'Honor Among Thieves Crit Rate',
 	labelTooltip: 'Number of crits other group members generate within 100 seconds',
+});
+
+export const pkSwap = InputHelpers.makeSpecOptionsBooleanInput<Spec.SpecRogue>({
+	fieldName: 'pkSwap',
+	label: 'PK Swap',
```
Comment (kayla-glick) [inline ui/tank_rogue/inputs.ts:18]: Same here

**Comment by emsimpson92** [comment]
Screenshot_20241126-111551
I'm going to address these changes as well

**Review by kayla-glick — APPROVED** [review]

**Pull request merged**